core/modules/arp_responder.h

```diff
@@ -0,0 +1,36 @@
+#ifndef BESS_MODULES_ARP_RESPONDER_H_
+#define BESS_MODULES_ARP_RESPONDER_H_
+
+#include <map>
+
+#include "../module.h"
+#include "../module_msg.pb.h"
+#include "../utils/endian.h"
+#include "../utils/ether.h"
+#include "../utils/ip.h"
+
+using bess::utils::Ethernet;
+using bess::utils::be32_t;
+
+struct arp_entry {
+  Ethernet::Address mac_addr;
+  be32_t ip_addr;
+  uint64_t time;
```

Comment:
Done, added that its timestamp, its currently not used but will be in future patches